packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -2056,6 +2056,26 @@ describe( 'Addition Settings UI', () => {
 			} )
 		);
 	} );
+
+	it( 'should show tooltip with full URL alongside filtered display', async () => {
+		const user = userEvent.setup();
+		const url = 'https://example.com';
```

Comment:
I think our test should be for a longer URL and probably one that includes a file format. That would add an extra layer of resilience to the test.

Reply:
Agreed, a longer URL would make sense here.

Reply:
Will do!